_posts/2017-03-03-Introducao-e-casos-de-uso-Map-Filter-e-Reduce.md

```diff
@@ -0,0 +1,201 @@
+---
+layout:     post
+title:      "Introdução e casos de uso: Map, Filter e Reduce."
+date:       2017-03-03 02:19:00
+author:     "Ezequiel França"
+header-img: "img/ezefranca/header.jpg"
+category:   "swift"
+---
+
+> Ezequiel França ([@ezefranca](https://twitter.com/ezefranca){:target="_blank"}) é desenvolvedor iOS, postador de zueiras no Slack e também entusiasta de Internet das Coisas e mundo Maker.
+
+
+Bem, eu geralmente gosto de escrever sobre o que me fez falta. Quando falamos em Map, Filter e Reduce, principalmente para um desenvolvedor junior é como se estivéssimos falando em uma bicho de sete cabeças. Acredito que isso naturalmente seja resultado de como programção é ensinada, tópicos como programação funcional são típicos de cursos de Ciência ou Engenharia da Computação (ou Matemática), por terem um rigor matemático maior. Entretanto para utilizar muitos destes recursos você não precisa ter estudado Cálculo IV ou ter muitos e muitos anos de programação para entender. 
```

Comment:
O sentido da última frase está estranho. 
`Entretanto para utilizar muitos destes recursos você não precisa ter estudado Cálculo IV ou ter muitos e muitos anos de programação para entender. `  Não precisa ter estudado para `utilizar` os recursos, ou para `entender`? Sugiro retirar o `para entender` do final. Se achar interessante, pode jogar os dois no início da frase: `Entretanto para entender e utilizar muitos destes recursos...`